definitions/output/crawl/pages.js

```diff
@@ -52,23 +52,91 @@ publish('pages', {
 DELETE FROM ${ctx.self()}
 WHERE date = '${constants.currentMonth}' AND
   client = 'desktop';
-`).query(ctx => `
+
+INSERT INTO ${ctx.self()}
 SELECT
   *
 FROM ${ctx.ref('crawl_staging', 'pages')}
 WHERE date = '${constants.currentMonth}' AND
   client = 'desktop'
-  ${constants.devRankFilter}
-`).postOps(ctx => `
+  ${constants.devRankFilter};
+
 DELETE FROM ${ctx.self()}
 WHERE date = '${constants.currentMonth}' AND
   client = 'mobile';
-
-INSERT INTO ${ctx.self()}
+`).query(ctx => `
 SELECT
   *
 FROM ${ctx.ref('crawl_staging', 'pages')}
 WHERE date = '${constants.currentMonth}' AND
   client = 'mobile'
   ${constants.devRankFilter}
+`).postOps(ctx => `
+CREATE TEMP TABLE technologies_cleaned AS (
+  WITH wappalyzer AS (
+    SELECT DISTINCT
+      name AS technology,
+      categories
+    FROM ${ctx.ref('wappalyzer', 'technologies')}
+  ), pages AS (
```

Comment:
Nit, I find it really hard to read when CTEs are merged like this.

```suggestion
  ),
  
  pages AS (
```

Reply:
I'm trying to adjust 😄